filters/include/pcl/filters/spherical_voxel_grid.h

```diff
@@ -0,0 +1,357 @@
+/*
+ * Software License Agreement (BSD License)
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2009, Willow Garage, Inc.
```

Comment:
I don't think we need Willow here. Your contribution is brand new code, even if you were "inspired" by the voxel grid implementation. As such, it should be copyrighted for Open Perception 2018.